tests/unit/src/app/kbv/controllers/self-assessment-payment-question.test.js

```diff
@@ -43,8 +43,12 @@ describe("self-assessment-payment-question controller", () => {
         req.session.question.questionKey = questionKey;
         req.form.values = {
           selfAssessmentPaymentDate: "2023-02-22",
-          selfAssessmentPaymentAmount: 2000.22,
+          selfAssessmentPaymentAmount: "2000.22",
         };
+        const userInput = JSON.stringify({
+          amount: new Number(req.form.values.selfAssessmentPaymentAmount),
```

Comment:
Can you try 
`amount: parsFloat(req.form.values.selfAssessmentPaymentAmount),`

and see if it behaves as expected

Reply:
Did as above and it works